### ISSUE TITLE
vlc/livestreamer wont open without twitch login

### ISSUE THREAD

**Issue by littlebigboss** [lindstrm/twitcha#22]
hi

i used your "old" twitcha tool at the beginning of the year and i wanted to say ...
your "new" tool works and looks great ... i really like the grid-like interface where you can see all your live channels

BUT

a few hours ago i wanted to open a stream with livestreamer but it didnt work ... my pc / the tool was open all day ... now livestreamer will open the stream in vlc ONLY when im logged in to twitch (in your tool) ... i deleted livestreamer in the twitcha folder (appdata/roaming/twitcha) and redownloaded it but theres no change ... opening the stream in browser works without login

it worked fine for several days and i dont think i changed something important

OS: Win7 64bit

**Comment by lindstrm** [comment]
Thanks :)
Yes, i seem to get it aswell.. Will fix it for next update.

**Comment by lindstrm** [comment]
Looks like an issue for Livestreamer.

**Comment by mahousenshi** [comment]
I'm using the old version yet. I'm using a 32 bits machine yet. There's a way to update the Livestreamer on the older twitcha when they fix the problem?

And I think you should have a way to download the old version for old machines.

Thnaks for twitcha.

**Comment by lindstrm** [comment]
Yes, just replace the files in the Livestreamer folder located in %appdata%\Twitcha (iirc).

The old version is still available on https://github.com/lindstrm/twitcha/releases, might be hard to find though i guess.
